### DISCUSSION TITLE
woc

### DISCUSSION THREAD

**Discussion by yansai0v0** [share121/inter-knot#4950] [崩坏：星穹铁道]
![mmexport1722313474410](https://github.com/user-attachments/assets/8a5d2292-0a63-4a7a-9ba9-7f65820d0207)

**Comment by yansai0v0** [comment]
好耶

**Comment by Ht-715** [comment]
诶嘿

**Comment by xiaoliaoggg** [comment]
有点小抽象∑(O_O；)🙂
